src/main/java/bio/terra/cli/app/utils/LocalProcessLauncher.java

```diff
@@ -76,6 +77,12 @@ public void launchProcess(
     }
   }
 
+  public Process launchSilentProcess(List<String> command, Map<String, String> envVars, Path workingDirectory) {
+    launchProcess(command, null, null);
```

Comment:
i don't follow. why passing in envVars and workingDirectory but not using them?

Reply:
Oops thanks. I'm removing this method since its unneeded abstraction, using `launchProcess` and `waitForTerminate` instead.